.husky/_/husky.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
```

Comment:
I think this is from the v5 upgrade. This file should get deleted.

Reply:
oh, mixed it up with the one under .git folder!

Reply:
fixed